# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix errno for secondary threads on Alpha
# DP: Author: Daniel Jacobowitz <dan@debian.org>
# DP: Upstream status: Pending
# DP: Status Details: Submitted Jan 8, 2003
# DP: Date: Jan 8, 2003, updated 2003-03-23 by gotom.

--- sysdeps/unix/alpha/sysdep.S	2003-03-23 16:01:02.000000000 +0900
+++ sysdeps/unix/alpha/sysdep.S.new	2003-03-23 16:02:50.000000000 +0900
@@ -96,14 +96,10 @@
 	PROLOGUE
 
 	/* Find our per-thread errno address  */
-#if defined PIC && !defined IS_IN_librt
-	bsr	ra, __errno_location	!samegp
-#else
 	jsr	ra, __errno_location
 #ifndef GPSAVEREG
 	ldgp	gp, 0(ra)
 #endif
-#endif
 
 	/* Store the error value.  */
 	ldq	t0, 8(sp)
